.flake8

```diff
@@ -0,0 +1,5 @@
+    [flake8]
+    ignore = E203, E266, E501, W503, F403, F401
+    max-line-length = 88
```

Comment:
Le consensus actuel sur la longueur max d'une ligne tourne autour de `119` caractères.

Je suggère que nous adoptions cette limite aussi.

[Un peu de rationnel ici](http://docs.bigchaindb.com/projects/contributing/en/latest/cross-project-policies/python-style-guide.html#maximum-line-length
) :

> We discussed this at length, and it seems that the consensus is: try to keep line lengths less than 79/72 characters, unless you have a special situation where longer lines would improve readability. 

[Et ici](https://code.djangoproject.com/ticket/23395) :

> Perhaps outside the scope of this ticket, but I'd like to propose adopting a max line length of 120 as that's the width of code review in GitHub


[Et un peu de doctrine ici](https://github.com/mozilla/foundation.mozilla.org/issues/940) :

> "Amen!" to that, the 80 character limit is an anachronism and forces you to write worse code in many situations, and not better code in any situation.

On peut le passer à Black en option aussi :

```
black --line-length 119
```

Reply:
Ni pour ni contre. 
Pour ma part, je ne mets pas de limitation de lignes, sauf si ça devient trop le souk.
Mais 80, c'est effectivement court.

Reply:
En général je serais ok mais là je vais mettre mon véto en raison de mes handicaps visuels personnels.

Je travaille en basse réso 1024x768 et des lignes plus longues me demanderaient de scroller tout le temps vers la droite. No way.

Reply:
Comment tu fais les code review Github qui font 120 caractères ?

Reply:
Je touche du bois, comme tous mes projets récents ont des lignes courtes, c'est très rare que j'ai besoin de scroller vers la droite sur les PR github, mais cela arrive.

Reply:
Je te propose de tenter le coup à 119. Si tu constates que c'est handicapant pour toi, on fera machine arrière mais au moins on aura testé. Qu'en penses-tu ?

Reply:
Voici un screenshot de mon écran complet:

![image](https://user-images.githubusercontent.com/10533583/76301575-295a0200-62c7-11ea-89b7-d4cf9af9ace7.png)

je dois souvent virer la sidebar quand les lignes sont un peu longues

![image](https://user-images.githubusercontent.com/10533583/76301623-38d94b00-62c7-11ea-9bc1-7ea3ecf3338f.png)

si on passe de 80 as 120 ça va être méga galère pour moi au quotidien, d'où mon véto.

Reply:
Screenshots plus complets: 

sans sidebar

![image](https://user-images.githubusercontent.com/10533583/76301840-a08f9600-62c7-11ea-923b-dbdbaff7e1bf.png)

avec sidebar

![image](https://user-images.githubusercontent.com/10533583/76301856-a8e7d100-62c7-11ea-834c-8ad2adece6d9.png)

Reply:
Ok @vgrange tout le monde s'adapte à ton setup alors, restons sur 88

Reply:
Ce test ne servirait à rien vu que je sais déjà que ce serait handicapant pour moi. Cf mes screenshots ci-dessus 80 est déjà juste. D'où mon véto. Je comprends pas l'intérêt de continuer cette conversation.

Reply:
On en reparlera avec plaisir le jour de la sortie de l'Oculus Go 2 et/ou Oculus Quest 2 qui auront surement une bien meilleure résolution qui me permettra de comfortablement utiliser une plus haute réso! Tu peux pas savoir comme j'ai hâte de ce jour! J'attends aussi d'autre évolutions tech (notamment le Hololens 2).

Reply:
Tu sais qu'il suffit de configurer ton éditeur de code si tu veux moins que 119 et de laisser 119 pour les autres hein ;)

Reply:
Très bon point. Sur Sublime c'est View > Word Wrap.

![image](https://user-images.githubusercontent.com/10533583/76302956-91a9e300-62c9-11ea-88ed-f0721d65e835.png)

![image](https://user-images.githubusercontent.com/10533583/76302983-9ec6d200-62c9-11ea-82f4-eb0e6dfc3d5d.png)

Je vais creuser.

On va peut-être finalement passer à 119 :-D à suivre...

Reply:
Tu peux le configurer dans tes settings @vgrange : https://forum.sublimetext.com/t/how-can-i-set-word-wrap-turned-on-by-default/573/11